spm_funcs.py

```diff
@@ -54,8 +54,14 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    spm_vals=[]
+    img=nib.load(fname)  
+    #- Get the data
+    data=img.get_fdata()
+    #- Calculate the SPM global value for each volume.
+    for i in range(data.shape[-1]):
+        spm_vals.append (spm_global(data[:,:,:,i]))
+    return spm_vals  # Return the result.
```

Comment:
Notice that the docstring above says this should be an array - you might consider converting the list to an array here.